src/extension.ts

```diff
@@ -895,7 +895,19 @@ export function findKindNameOrPrompt(resourceKinds: kuberesources.ResourceKind[]
 }
 
 function promptKindName(resourceKinds: kuberesources.ResourceKind[], descriptionVerb, opts, handler) {
-    vscode.window.showInputBox({ prompt: "What resource do you want to " + descriptionVerb + "?", placeHolder: 'Empty string to be prompted' }).then((resource) => {
+    let placeHolder: string = 'Empty string to be prompted';
+    let prompt: string = "What resource do you want to " + descriptionVerb + "?";
+    if (opts) {
+        // see if the options has 'placeHolder' string, if so use it instead of default one
```

Comment:
I would drop this comment (and similarly for prompt) - the code is clear and I don't think it needs any additional explanation through the comment.

Reply:
sorry, i added for my clarity. forgot to remove